### ISSUE TITLE
Can you upload your source code for evaluation? Thanks

### ISSUE THREAD

**Issue by jdm4pku** [TempUpload/FLofChatGPT#1]

**Comment by appmlk** [comment]
Of course, this part of the code is a bit messy, please give me some time to prepare. Unfortunately, the code for interacting with ChatGPT Interface is no longer valid (because ChatGPT Interface has been upgraded many times since then).
